<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMEManagerWidget</class>
 <widget class="QWidget" name="FMEManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Optimal Value Cluster Graph</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_5">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../DsgTools/icons/histogram.png</normaloff>../../../../DsgTools/icons/histogram.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input Layer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="toolButton_4">
     <property name="text">
      <string>&lt;&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="4">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>   &gt;  </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButton_1">
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="4">
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="toolButton_3">
     <property name="text">
      <string>   &lt;  </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
